CHANGELOG.md

```diff
@@ -16,6 +16,9 @@
 - [usd#1784](https://github.com/Autodesk/arnold-usd/issues/1784) - The aov layer name is now correctly taken into account when rendering exrs with husk and using the arnold productType.
 
 
+### Build
```

Comment:
shouldn't it be in the Features instead ?

Reply:
changed